common/test/acceptance/tests/test_cohorted_courseware.py

```diff
@@ -48,6 +54,26 @@ def setUp(self, is_staff=True):
             self.browser, username=self.cohort_b_student_username, email=self.cohort_b_student_email, no_login=True
         ).visit()
 
+        # Create a Verified Student
+        self.cohort_verified_student_username = "cohort_verified_student"
+        self.cohort_verified_student_email = "cohort_verified_student@example.com"
+        StudioAutoAuthPage(
+            self.browser,
+            username=self.cohort_verified_student_username,
+            email=self.cohort_verified_student_email,
+            no_login=True
+        ).visit()
+
+        # Create audit student
+        self.cohort_audit_student_username = "cohort_audit_student"
```

Comment:
As mentioned before, let's make sure all learners are in a track so that the test more closely mimics reality.